ClientInfo.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DNWS
+{
+  class ClientInfo : IPlugin
+  {
+    protected static Dictionary<String, int> statDictionary = null;
```

Comment:
Why we need statDictionary here,

### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!
### Why!